geoapps/io/Gravity/constants.py

```diff
@@ -0,0 +1,1022 @@
+
+import numpy as np
+from geoh5py.workspace import Workspace
+
+required_parameters = [
+    "inversion_type",
+]
+defaults = {}
+
+default_ui_json = {
+    "inversion_type": {
+        "default": "gravity",
+        "visible": False,
+        "enabled": False,
+        "value": "gravity",
+    },
+    "forward_only": {
+        "default": False,
+        "main": True,
+        "label": "forward model only?",
+        "value": False,
+    },
+    "topography_object": {
+        "default": None,
+        "main": True,
+        "group": "Topography",
+        "label": "Object",
+        "meshType": [
+            "{202C5DB1-A56D-4004-9CAD-BAAFD8899406}",
+            "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}",
+            "{F26FEBA3-ADED-494B-B9E9-B2BBCBE298E1}",
+            "{48F5054A-1C5C-4CA4-9048-80F36DC60A06}",
+        ],
+        "value": None,
+    },
+    "topography": {
+        "association": "Vertex",
+        "dataType": "Float",
+        "default": None,
+        "group": "Topography",
+        "main": True,
+        "dependency": "forward_only",
+        "dependencyType": "hide",
+        "isValue": False,
+        "label": "Elevation",
+        "parent": "topography_object",
+        "property": None,
+        "value": 0.0,
+    },
+    "data_object": {
+        "default": None,
+        "main": True,
+        "group": "Receivers",
+        "label": "Object",
+        "meshType": [
+            "{202C5DB1-A56D-4004-9CAD-BAAFD8899406}",
+            "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}",
+            "{F26FEBA3-ADED-494B-B9E9-B2BBCBE298E1}",
+            "{48F5054A-1C5C-4CA4-9048-80F36DC60A06}",
+        ],
+        "value": None,
+    },
+    "gz_channel": {
+        "association": "Cell",
+        "dataType": "Float",
+        "default": None,
+        "group": "Data",
+        "main": True,
+        "dependency": "forward_only",
+        "dependencyType": "hide",
+        "label": "Channel",
+        "parent": "data_object",
+        "value": None,
+    },
+    "gz_uncertainty": {
+        "association": "Cell",
+        "dataType": "Float",
+        "default": 1e-2,
+        "group": "Data",
+        "main": True,
+        "dependency": "forward_only",
+        "dependencyType": "hide",
+        "isValue": True,
+        "label": "Uncertainty",
+        "parent": "data_object",
+        "property": None,
+        "value": 1e-2,
+    },
+    "starting_model_object": {
+        "default": None,
+        "group": "Starting Model",
+        "meshType": [
+            "{202C5DB1-A56D-4004-9CAD-BAAFD8899406}",
+            "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}",
+            "{F26FEBA3-ADED-494B-B9E9-B2BBCBE298E1}",
+            "{48F5054A-1C5C-4CA4-9048-80F36DC60A06}",
+        ],
+        "label": "starting model object",
+        "value": None,
+    },
+    "starting_model": {
+        "association": "Cell",
+        "dataType": "Float",
+        "default": None,
+        "group": "Starting Model",
+        "isValue": False,
```

Comment:
Inversion crashes if the user doesn't click on value. Should default to "isValue"=true, with default value.

Reply:
Gotcha, fixed up.